backend/curriculum_tracking/tests/test_api_views.py

```diff
@@ -447,6 +457,98 @@ def test_set_project_link(self):
         project.refresh_from_db()
         self.assertEqual(project.link_submission, link_2)
 
+    def test_get_number_of_times_card_moved_to_review_feedback(self):
+        recruit = UserFactory()
```

Comment:
extremely minor, but this recruit item does not appear to be used, so can be deleted